src/main/java/com/nicusa/controller/SecurityController.java

```diff
@@ -52,6 +53,11 @@ public String signup(WebRequest request) {
     userProfile.setName(socialUserProfile.getName());
     userProfile.setEmailAddress(socialUserProfile.getEmail());
     entityManager.persist(userProfile);
+    Portfolio portfolio = new Portfolio();
```

Comment:
Lets move this into the UserProfile domain object and set the relationship between UserProfile and Portfolio to cascade all.